Formula/wb32-dfu-updater_cli.rb

```diff
@@ -1,5 +1,5 @@
 class Wb32DfuUpdaterCli < Formula
-  desc "USB programmer"
+  desc "Wb32-dfu-updater_cli is a tool for downloading and uploading firmware to/from devices."
```

Comment:
Style checks dictate it should be more like this:
```suggestion
  desc "Tool for downloading and uploading firmware to/from devices"
```